### PR TITLE
Bug fix for call filtering.

### PR DESCRIPTION
**Pull request by deflaux**

### CODE REVIEW

File: Java/PlatinumGenomes-variant-transformation/src/test/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentDataTest.java [hunk 1]
```diff
@@ -63,17 +66,40 @@ public void testFilterVariantCallsFn() {
         ListValue.newBuilder()
             .addValues(Value.newBuilder().setStringValue("VQSRTrancheSNP99.90to100.00").build())
```
Comment (pgrosu) [inline Java/PlatinumGenomes-variant-transformation/src/test/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentDataTest.java]: Just out of curiosity, where in the API is the tranche level of a variant used?  I didn't think recalibration is being performed by the API and doesn't seem to affect this test.

Comment (deflaux) [inline Java/PlatinumGenomes-variant-transformation/src/test/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentDataTest.java]: This test case is just trying to show a non-passing filter value that might occur in data imported to the API.

Comment (pgrosu) [inline Java/PlatinumGenomes-variant-transformation/src/test/java/com/google/cloud/genomics/examples/TransformNonVariantSegmentDataTest.java]: That's fine, but the issue is that you are developing the test after the implementation.  You might say *"Eh Paul, everything is fine.  Why is that important?  It works in passing my tests."*  Well, the thing is that it is better to have a collection of VCF files upfront that fulfill the edge-case and satisfiability-criteria.  As you evolve the fundamental definitions and the implementations for the cloud - of a variant, annotation, etc - to match what users are encountering these days, you want users to have a seemless experience.  As folks continue to build on top of the Google API to generate the tools - especially as utils-java is core the whole API - they will assume the tool will perform as expected.  Otherwise, you might encounter unexpected and unmanageable scope creeps down the line that would overwhelm any migration/data-update efforts, for anything already loaded into GoogleGenomics and/or user-driven feature expansion efforts.
